cmd/_integration-tests/transport/transport-test.proto

```diff
@@ -27,8 +27,15 @@ service TransportPermutations {
       body: "*"
     };
   }
+  rpc ErrorRPC (Empty) returns (Empty) {
+    option (google.api.http) = {
+      get: "/error"
+    };
+  }
 }
 
+message Empty {}
```

Comment:
Trivia: Did you know that [Protobuf defines an `Empty` type](https://github.com/google/protobuf/blob/master/src/google/protobuf/empty.proto)? Doesn't really matter, but I have seen a bunch of empty redefinitions in our MV protobuf stuff.